.ci/containers/downstream-builder/Dockerfile

```diff
@@ -64,5 +64,8 @@ RUN git config --global user.name "Modular Magician"
 RUN git config --global user.email "magic-modules@google.com"
 
 ADD generate_downstream.sh /generate_downstream.sh
+ADD test_tpgc.sh /test_tpgc.sh
```

Comment:
I'm not a huge fan of combining all the use cases into a single docker image with different endpoints. Can we just keep the go version in sync across these images?

Reply:
maybe we could have a base image that has any shared tooling / dependencies and then have all the steps inherit from that instead of directly from golang images?

Reply:
Yeah, that's reasonable. I think the amount of shared config between these containers is only the go version, which is why I hesitate to combine them

Reply:
It would specifically be nice to have access to jq on all the containers because I'm currently having to hand-code JSON strings in bash 😂  (for https://github.com/hashicorp/terraform-provider-google/issues/9146)